.github/workflows/conda-package.yml

```diff
@@ -114,24 +114,25 @@ jobs:
 
       - uses: actions/checkout@v3
         with:
-          path: numba-dppy
+          path: numba-dpex
 
       - name: Setup conda-build
-        uses: ./numba-dppy/.github/actions/setup-conda-build
+        uses: ./numba-dpex/.github/actions/setup-conda-build
 
       - name: Create conda channel
         run: |
           mkdir -p $GITHUB_WORKSPACE/channel/linux-64
           mv ${PACKAGE_NAME}-*.tar.bz2 $GITHUB_WORKSPACE/channel/linux-64
           conda index $GITHUB_WORKSPACE/channel
           # Test channel
-          conda search $PACKAGE_NAME -c $GITHUB_WORKSPACE/channel --override-channels
+          conda search $PACKAGE_NAME -c $GITHUB_WORKSPACE/channel --override-channels --info
 
-      - name: Install numba-dppy
+      - name: Install numba-dpex
         env:
-          CHANNELS: -c intel -c defaults -c numba -c numba/label/dev -c dppy/label/dev --override-channels
+          CHANNELS: -c dppy/label/dev -c intel -c defaults -c numba -c numba/label/dev  --override-channels
```

Comment:
I do not sure that this reordering of channels will give correct result.

Reply:
reverted